base_plugins/hdfs/main.py

```diff
@@ -0,0 +1,18 @@
+from aztk.plugins import PluginDefinition, PluginPort, PluginRunTarget
+
+
+def definition():
+    return PluginDefinition(
+        name="hdfs",
+        ports=[
+            PluginPort(
+                internal=8020,
```

Comment:
Need to also open 50010, 50020, 50070, & 50075

Reply:
as internal?